pennylane/qchem/hamiltonian.py

```diff
@@ -550,6 +550,10 @@ def _molecular_hamiltonian(
         )
 
         requires_grad = args is not None
+        use_jax = any(qml.math.get_interface(x) == "jax" for x in [coordinates, alpha, coeff])
```

Comment:
In Hamiltonian? I don't think so. The warning will be caught in `molecule.py` since that's where the information is originally coming from. If you put it in Hamiltonian you'll just get the same warning twice.

Reply:
Cool cool. I forgot we are passing `args` directly via `Molecule`.